app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
+end
```

Comment:
- [ ] This model shouldn't be empty. A user should have many `posts/likes/comments`. For example, `has_many :posts, foreign_key: :author_id`